<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_Start">
       <property name="text">
        <string>开始</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_SetTime">
       <property name="text">
        <string>设置时间</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>551</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>当前时间（时：分：秒）</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdNumber_Hour"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_Minute"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_Second"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Reduce">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>347</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间已过去：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
